apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/deprecated/MapAssertionSamples.kt

```diff
@@ -0,0 +1,252 @@
+                KeyValue(1, { // subject inside this block is of type String (actually "a")
+                    toBe("a")
+                })
+            )
+
+        fails {
+            expect(mapOf(1 to "a"))
+                .contains(
+                    KeyValue(1, {   // subject inside this block is of type String (actually "a")
+                        toBe("b") // fails because "a" is not equal to "b"
+                    })
+                )
+        }
+    }
+
+    @Test
+    fun containsOnlyKeyValue() {
+        expect(mapOf(1 to "a"))
+            .containsOnly(
+                KeyValue(1, {   // subject inside this block is of type String (actually "a")
+                    toBe("a")
+                })
+            )
+
+        fails {
+            expect(mapOf(1 to "a", 1 to "b"))
+                .containsOnly(
+                    KeyValue(1, {   // subject inside this block is of type String (actually "a")
+                        toBe("a")   // fails because the map also contains Pair<1,"b">
+                    })
+                )
+        }
+    }
+
+    @Test
+    fun containsEntriesOf() {
+        expect(mapOf(1 to "a")).containsEntriesOf(listOf(1 to "a"))
+
+        fails {
+            expect(mapOf(1 to "a")).containsEntriesOf(listOf(1 to "b")) // fails because the map not contains <1,"b">
+        }
+    }
+
+    @Test
+    fun containsOnlyEntriesOf() {
+        expect(mapOf(1 to "a")).containsOnlyEntriesOf(listOf(1 to "a"))
+
+        fails {
+            expect(mapOf(1 to "a", 1 to "b"))
+                .containsOnlyEntriesOf(listOf(1 to "a")) // fails because the map not only contains <1,"a">
+        }
+    }
+
+    @Test
+    fun containsKey() {
+        expect(mapOf(1 to "a")).containsKey(1)
+
+        fails {
+            expect(mapOf(1 to "a")).containsKey(2) // fails because the map does not contain a key that equals 2
+        }
+    }
+
+    @Test
+    fun containsNotKey() {
+        expect(mapOf(1 to "a")).containsNotKey(2)
+
+        fails {
+            expect(mapOf(1 to "a")).containsNotKey(1)   // fails because the map contains key that equals 1
+        }
+    }
+
+    @Test
+    fun getExisting() {
+        expect(mapOf(1 to "a"))
+            .getExisting(1) // subject is of type String (actually "a")
+            .toBe("a")
+
+        fails {
+            expect(mapOf(1 to "a"))
+                .getExisting(1)  // subject is of type String (actually "a")
+                .toBe("b")  // fails because "a" is not equal to "b"
+        }
+    }
+
+    @Test
+    fun getExistingKeyAssertion() {
+        expect(mapOf(1 to "a"))
+            .getExisting(1, {   // subject inside this block is of type String (actually "a")
+                toBe("a")
+            })
+
+        fails {
+            expect(mapOf(1 to "a"))
+                .getExisting(1, {   // subject inside this block is of type String (actually "a")
+                    toBe("b")   // fails because "a" is not equal to "b"
```

Comment:
detail, would look nicer if it is aligned
```suggestion
                    toBe("b")      // fails because "a" is not equal to "b"
```